x-pack/plugins/maps/public/classes/styles/vector/properties/test_util.ts

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Can you move to `__tests__` folder so the file is not included in the production bundle.

excluded files from build
https://github.com/elastic/kibana/blob/512ab5beb380650c0a65543199cec2a77281db67/x-pack/.kibana-plugin-helpers.json#L7-L34